GOVERNANCE.md

```diff
@@ -268,9 +268,12 @@ In the case of abstention, an approver will not be included in the voting approv
 group when deciding the majority. In the case of a tie, the issue will not be considered
 to have passed.
 
-Most votes require a simple majority of all voting approvers to succeed. Approvers
-can be removed by a 2/3 majority vote of all voting approvers, and changes to this
-Governance require a 2/3 vote of all approvers.
+Most votes require a simple majority of all voting approvers to succeed.
+Approvers can be removed by a 2/3 majority vote of all voting approvers,
+and changes to this Governance require a 2/3 vote of all approvers. The
+approvers will make every reasonable effort to make sure that any removal
+or amendment to the Governance be affirmed with votes by employees of
+more than one organization.
```

Comment:
I agree with @haircommander that any kind of change like this one should be open for at least a week, giving enough time and ensuring that "The approvers will make every reasonable effort to amke sure that any removal or amendment to the Governance be affirmed with the votes by employees of mnore than one organization".

Regardless of this one being merged or not, I'm 100% in favour of the wording, as it's mostly wordsmith here and the project's always been open (and actually pushing forward to) having a good mix of folks working on different companies.

Thanks, @haircommander, lgtm!